testing/kfctl/kf_is_ready_test.py

```diff
@@ -145,6 +149,52 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio, app_path):
     logging.info("Verifying that deployment %s started...", deployment_name)
     util.wait_for_deployment(api_client, knative_namespace, deployment_name, 10)
 
+
+def test_gcp_access(namespace, app_path, app_name, project_id):
```

Comment:
Why do we need app_name and project_id? Can't we get those from app_path?

You should also add a function argument for the record_xml_attribute
Then call 

```
 util.set_pytest_junit(record_xml_attribute, "test_gcp_access")
```

This will set some xml attributes in the junit file that will make it easier to group tests based on workflow.

Reply:
Updated to get rid of app_name.
Can extract project once kfdef version migration is done.